loadtest/user/userentity/websocket.go

```diff
@@ -33,7 +76,11 @@ func (ue *UserEntity) handlePostEvent(ev *model.WebSocketEvent) error {
 
 	switch ev.EventType() {
 	case model.WEBSOCKET_EVENT_POSTED, model.WEBSOCKET_EVENT_POST_EDITED:
-		return ue.store.SetPost(post)
+		if currentChannel, err := ue.store.CurrentChannel(); err == nil && currentChannel.Id == post.ChannelId {
```

Comment:
I will prefer if we use the long form for this. The `err == nil && currentChannel.Id == post.ChannelId` in the same line as the if condition is getting a bit too long to be readable. :)